PlannTer/UI/Inputs/TextInput.swift

```diff
@@ -0,0 +1,33 @@
+import SwiftUI
+
+struct TextInput: View {
+    let title: String
+    let prompt: String
+
+     @State private var inputText: String = ""
+    
+    var body: some View {
+        VStack(alignment: .leading) {
+            Text(title)
+                .font(.note)
+            TextField(prompt, text: $inputText)
+                .frame(height: 50)
+                .padding(.horizontal, 20)
+                .background(Color.white.opacity(0.3))
+                .overlay(
+                    Rectangle()
+                        .frame(height: 1)
+                        .foregroundColor(.primaryText),
+                    alignment: .bottom
+                )
+                .cornerRadius(3)
+                .font(.subheadline)
```

Comment:
I'm not sure if that's what you meant (it used to be `.subtitle`)

Reply:
I meant secondary text